protocol/x/ratelimit/types/keys.go

```diff
@@ -27,3 +31,16 @@ const ()
 func GetPendingSendPacketKey(channelId string, sequenceNumber uint64) []byte {
 	return []byte(fmt.Sprintf("%s_%d", channelId, sequenceNumber))
 }
+
+func SplitPendingSendPacketKey(key []byte) (channelId string, sequenceNumber uint64) {
+	parts := bytes.Split(key, []byte("_"))
+	if len(parts) != 2 {
+		panic(fmt.Sprintf("unexpected key format: %s", key))
```

Comment:
What does user see if we panic here? I assume nothing or some internal error? 

Let's:
- Log an error here so we could see this on server side
- Also return this error so the user can see this error as well (easier for troubleshooting)

The error string could also be more helpful (e.g. "unexpected PendingSendPacket key format...")